org/wildfly/ee/concurrent/scheduledexecutor/capability.adoc

```diff
@@ -0,0 +1,34 @@
+NAME: org.wildfly.ee.concurrent.scheduledexecutor
+
+DESCRIPTION: Provides a ManagedScheduledExecutorService, as specified by Concurrency Utilities for Java EE (JSR 236).
+
+REGISTERED BY:
+
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: true
+
+PRIVATE: true
+
+SUPPORTS RUNTIME ONLY: false
+
+SERVICE PROVIDED:
+
+  CLASS: org.glassfish.enterprise.concurrent.ManagedScheduledExecutorServiceAdapter
```

Comment:
Can we not use javax.enterprise.concurrent.ManagedScheduledExecutorService for the service type? Generally, we should avoid exposing specific implementation classes via the capability contract.

Reply:
Resolving, as capability is private.